conda-recipes/meta.yaml

```diff
@@ -17,25 +21,23 @@ source:
   path: ../
 
 build:
-  number: 0
+  number: 3
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
-    - python=3.7
+    - python={{ python_version }}
     - setuptools
 
   run:
-    - python=3.7
-    - cc3d
+    - python={{ python_version }}
+    - cc3d-player5={{ version }}
     - chardet
-    - psutil
-    - pyqt
-    - pyqtwebkit
-    - qscintilla2
+    - pyqtwebkit # [not osx]
```

Comment:
Is there an issue with this package on OSX? Not critical, since it's only used to launch the in-app documentation. Just curious.

Reply:
We are using slightly older PyQt on OSX and the composition of Qt suite is slightly different there. This is the only reason